tests/lib/config.js

```diff
@@ -249,8 +249,8 @@ describe("Config", () => {
                     configHelper.findLocalConfigFiles(getFakeFixturePath("broken"))
                 );
 
-            assert.equal(actual.length, 1);
-            assert.equal(actual, expected);
+            assert.strictEqual(actual.length, 1);
+            assert.deepEqual(actual, [expected]);
```

Comment:
Given that there's a length assertion above this line, we could assert for strict equality as follows:

```js
assert.strictEqual(actual[0], expected);
```

Reply:
Actually, it looks like [`assert.deepStrictEqual`](https://nodejs.org/api/assert.html#assert_assert_deepstrictequal_actual_expected_message) is supported on all necessary Node versions, so maybe we should switch to using that as well.

Reply:
I'd be in favor of that for sure. Maybe we could add `assert.deepEqual` to no-restricted-properties as well?